### DISCUSSION TITLE
Octo Integration

### DISCUSSION THREAD

**Discussion by joshmedeski** [dlvhdr/gh-dash#286] [Ideas]
Hey @dlvhdr!

I think it was be great to add support for https://github.com/pwntester/octo.nvim

To open a PR with Octo, you just pass the pull request's URL to the Octo command

```
:Octo https://github.com/repo/pulls/{num}
```

I'm thinking of overwriting `o` to open with Octo instead of the browser.

What do you think? I'm happy to contribute but I'm not familiar with gh-dash's codebase yet.

**Comment by dlvhdr** [comment]
Hey @joshmedeski!
There's actually a way to configure custom keybinds to do just that. I've mapped C to open Octo and this workflow is *chefs kiss*.

Check out my config in my dotfiles repo or this video: https://youtu.be/0VbWVNWeo7M

**Reply by joshmedeski** [reply]
Thanks, this is perfect. I'll check it out!
